web/impact/impact/v1/views/industry_list_view.py

```diff
@@ -2,10 +2,14 @@
 # Copyright (c) 2017 MassChallenge, Inc.
 
 from impact.v1.views.list_view import ListView
-from impact.v1.helpers import IndustryHelper
-from impact.v1.views.industry_detail_view import IndustryDetailView
+from impact.v1.helpers import (
+    INDUSTRY_FIELDS,
+    IndustryHelper,
+)
 
 
 class IndustryListView(ListView):
     helper_class = IndustryHelper
-    METADATA_ACTIONS = IndustryDetailView.METADATA_ACTIONS
+
```

Comment:
Shouldn't IndustryListView inherit from ImpactView as well?

Reply:
It does.  impact.v1.views.list_view.ListView inherits from BaseListView & AutoMetadataMixin and BaseListView inherits from ImpactView.

I suspect you are expecting ListView to refer to `django.views.generic.list.ListView`.  That's fair, but should be handled in a different ticket.

Reply:
Did not trace back from BaseListView. Fair enough.